docs/src/developers_guide/release.rst

```diff
@@ -255,6 +257,8 @@ Post Release Steps
    `Read The Docs`_ to ensure that the appropriate versions are ``Active``
    and/or ``Hidden``. To do this ``Edit`` the appropriate version e.g.,
    see `Editing v3.0.0rc0`_ (must be logged into Read the Docs).
+#. Make a new `latest.rst` from `latest.rst.template` and update the include
```

Comment:
should this be using double single quotes? (same on next line)

Reply:
[aab13c4](https://github.com/SciTools/iris/pull/4727/commits/aab13c401ef69f3f359cbbcbfd9fbac11acf63ae)